src/sdsc/version.py

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2015-2016 SUSE Linux GmbH
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+#
+
+__version__ = "2016.06.99.1"
```

Comment:
It's a bit a matter of taste. However, IMHO I think it's easier to parse, see `find_version()` function in `setup.py`. Could also have some benefits when using `bumpversion`.